db/seeds.rb

```diff
@@ -1,9 +1,324 @@
+puts "\n== Seeding the database with fake data created by Turing's team RubyHive. Thank you for your patience. ==\n\n"
```

Comment:
Thank you for creating seed data that will allow for easier localhost user testing in the future.